controls/nist_ocp4.yml

```diff
@@ -13530,13 +13530,27 @@ controls:
     https://issues.redhat.com/browse/CMP-478
   rules: []
 - id: SC-7(10)
-  status: pending
+  status: automated
```

Comment:
This seems like it would be a good candidate for a `partial` status (something to track in the future) since this appears to be a pretty hard check to fully automate (at least based on my interpretation of the control).

Reply:
I agree, I think we could only check for part of it